app/src/main/java/eu/faircode/netguard/Receiver.java

```diff
@@ -224,13 +224,9 @@ public static void upgrade(boolean initialized, Context context) {
                 Log.i(TAG, "Initializing sdk=" + Build.VERSION.SDK_INT);
                 editor.putBoolean("whitelist_wifi", false);
                 editor.putBoolean("whitelist_other", false);
-                if (Build.VERSION.SDK_INT == Build.VERSION_CODES.LOLLIPOP_MR1) {
-                    editor.putBoolean("filter", true); // Optional
-                    editor.putBoolean("filter_allowed", true); // Optional
-                }
             }
 
-            if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP) {
+            if (Build.VERSION.SDK_INT < VERSION_CODES.LOLLIPOP_MR1) {
```

Comment:
This should not be changed like this, since it will mandate filtering on Android 5.0, which is not the intention.